README.md

```diff
@@ -103,7 +103,7 @@ The output of spark-submit will be streamed to the console. Use the `--no-wait`
 If you decided not to tail the log when submiting the job or want to read it again you can use this command.
 
 ```bash
-azb spark app logs \
+azb spark cluster logs \
```

Comment:
also here the logs are a bit confusing are they will show the logs of an app you submitted with cluster submit. Maybe `azb spark cluster app-logs`